hathor/wallet/base_wallet.py

```diff
@@ -176,6 +177,9 @@ def handle_publish(self, key: HathorEvents, args: EventArguments) -> None:
         data = args.__dict__
         if key == HathorEvents.CONSENSUS_TX_UPDATE:
             self.on_tx_update(data['tx'])
+        elif key == HathorEvents.CONSENSUS_TX_REMOVED:
+            # we use the same method as above because a removed tx is also voided
+            self.on_tx_update(data['tx'])
```

Comment:
Should we assert here that this tx is voided?

Reply:
I guess it's a bit redundant, but no harm. Done in https://github.com/HathorNetwork/hathor-core/pull/1139/commits/6b95cb0b3e75f7551cb3c4f3f24f455de85ffdeb

Reply:
Thanks!